static/app/views/releases/detail/overview/releaseComparisonChart/releaseComparisonChartRow.tsx

```diff
@@ -0,0 +1,230 @@
+import {ReactNode} from 'react';
```

Comment:
This was extracted from `static/app/views/releases/detail/overview/releaseComparisonChart/index.tsx` because the file was getting quite large.